lib/locales/en.ts

```diff
@@ -517,12 +531,15 @@ export const locale: Locale = {
     ivan_merz: 'Blessed Ivan Merz',
     ivo_of_kermartin_priest: 'Saint Ivo, Priest',
     jacinta_marto_and_francisco_marto: 'Saints Jacinta and Francisco Marto',
+    jacques_jules_bonnaud_martyr_and_companions_martyrs: 'Blessed Jacques-Jules Bonnaud, and Companions, Martyrs',
```

Comment:
Omit duplicate `martyr` title from the key.

```suggestion
    jacques_jules_bonnaud_and_companions_martyrs: 'Blessed Jacques-Jules Bonnaud, and Companions, Martyrs',
```

Reply:
Key renamed to `jacques_jules_bonnaud_priest_and_companions_martyrs` + update En/Fr locales to add the Priest title.